grpc.go

```diff
@@ -668,6 +668,16 @@ func (a *agentGRPC) CreateContainer(ctx context.Context, req *pb.CreateContainer
 		return emptyResp, err
 	}
 
+	for _, e := range ociSpec.Process.Env {
+		if strings.HasPrefix(e, "NVIDIA_VISIBLE_DEVICES") {
```

Comment:
I think you should be a little bit more specific here,  how about using `"NVIDIA_VISIBLE_DEVICES="`?

Reply:
My negligence. Thanks for your patience.
To fix